### DISCUSSION TITLE
Update carousel_slider from 4.2.1 to 5.0.0

### DISCUSSION THREAD

**Discussion by OwaisKhatri** [wger-project/flutter#645] [Show and tell]
wger is not being compiled on using flutter 3.24.1 because of the below error:
`Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.`

The above error gets resolved after updating the carousel_slider from **4.2.1** to **5.0.0**.

**Comment by OwaisKhatri** [comment]
@rolandgeider++

**Comment by rolandgeider** [comment]
Fixed with https://github.com/wger-project/flutter/commit/15b6eb19b06aec62939e48f164fb2d88f87e9cc3 👍🏻
